.github/workflows/nodejs.yml

```diff
@@ -4,9 +4,6 @@ on:
   pull_request:
     branches:
       - master
-  push:
-    branches:
-      - master
```

Comment:
Technically, if every pull request is made towards `master` and we never push directly to it, this is just redundant. So, removed it.